lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -92,6 +92,149 @@ html.video-fullscreen {
       display: inline;
     }
   }
+
```

Comment:
Actually this is not merged to master, it's been merged to an intermediate branch (cdodge/timed-exams) where we are queuing up work on Timed Exams. When this cdodge/timed-exams, we'll open it up for wider review. Thanks for the note, though...

I think the challenge here is that we need this style for the timer bar which appears at the top of the page. I'm sure there is some Sass magic to both limit the style definition while also applying it to the timer bar as well.